frb_example/pure_dart/rust/src/api.rs

```diff
@@ -1320,3 +1320,15 @@ pub fn handle_type_alias_model(input: Id) -> TestModel {
         alias_struct: StructAlias { content: true },
     }
 }
+
+#[frb(dart_metadata = ("freezed"))]
+#[derive(Clone)]
+pub struct FreezedPlusFunction {
+    pub text: String,
+}
+
+impl FreezedPlusFunction {
+    pub fn as_string(&self) -> String {
```

Comment:
https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

```suggestion
    pub fn as_str(&self) -> &str {
```

or `pub fn inner(&self) -> String` etc